src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/LongRunningTriggerFunction.cs

```diff
@@ -0,0 +1,49 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Model;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
+
+public class LongRunningTriggerFunction(
+    ILogger<LongRunningTriggerFunction> logger)
+{
+    [Function(nameof(TriggerLongRunningOrchestration))]
+    public async Task<IActionResult> TriggerLongRunningOrchestration(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = nameof(TriggerLongRunningOrchestration))]
+        HttpRequest httpRequest,
+        [DurableClient] DurableTaskClient client,
+        CancellationToken cancellationToken)
+    {
+        var instanceId = Guid.NewGuid();
+
+        var durationSeconds =
+            httpRequest.GetRequestParamInt(paramName: "durationSeconds", 60);
```

Comment:
Minor, but might be clearer to declare this like `GetRequestParamInt("durationInSeconds", defaultValue: 60)`. From a PR perspective, it's unclear what 60 is supposed to mean.

Reply:
Yup good point, updated ta